### PR TITLE
The test "SessionNotStartedWhenCaretNotMappableIntoSubjectBuffer" cre…

### PR DESCRIPTION
**Pull request by rchande**
…ates an ITextView not managed by the test workspace.

To avoid leaking taggers, close that view at the end of the test.

@brettfo @tannergooding This should fix the test failure we're seeing.

### CODE REVIEW

File: src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb [hunk 1]
```diff
@@ -1563,6 +1563,7 @@ class C
                 state.CompletionCommandHandler.ExecuteCommand(New DeleteKeyCommandArgs(view, state.SubjectBuffer), Sub() editorOperations.Delete())
 
                 state.AssertNoCompletionSession()
+                view.Close()
```
Comment (brettfo) [inline src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb]: Maybe wrap the previous few lines in try/finally to ensure this always gets called?

**Comment by brettfo** [comment]
:shipit:

File: src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb [hunk 2]
```diff
@@ -1557,12 +1557,16 @@ class C
                 Dim projection = projectionBufferFactory.CreateProjectionBuffer(Nothing, New Object() {otherExposedSpan, subjectBufferExposedSpan}.ToList(), ProjectionBufferOptions.None)
 
                 Dim view = textViewFactory.CreateTextView(projection)
```
Comment (Pilchie) [inline src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb]: Maybe look for other calls to `EditorFactory.CreateView`.  I'm pretty sure there are a bunch of them.

Comment (rchande) [inline src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb]: @Pilchie  filed #5289

**Pull request merged**